components/button/button.css

```diff
@@ -61,7 +70,8 @@
 }
 
 .mt-button.mt-button-secondary {
```

Comment:
I think we can get away with not increasing the specificity for these other selectors. i.e.; `.mt-button-secondary` by itself should be enough. What do you think @JPorry ?

Reply:
Totally! I usually leave things with a specificity of one class, but there are some mistakes here and there that we need to detect and fix!
Thanks man!